pyproject.toml

```diff
@@ -78,6 +78,7 @@ dependencies = [
     "markitdown[pdf, docx, xlsx, xls, outlook, pptx]",
     "langchain-mcp-adapters",
     "atlassian-python-api>=4.0.7",
+    "audioop-lts>=0.2.2",
```

Comment:
Should the other libraries (twilio etc) not be pinned here as well?

Reply:
They could be but it's fine for their versions to be managed in the `uv.lock` file